Configurations/10-main.conf

```diff
@@ -1018,6 +1018,10 @@ my %targets = (
         shared_target    => "bsd-gcc-shared",
         shared_cflag     => "-fPIC",
     },
+    "NetBSD-generic32" => {
+	inherit_from	=> [ "BSD-generic32" ],
+	ex_libs		=> add(threads("-latomic")),
```

Comment:
It looks like something is wrong here with the indentation?

Reply:
I submitted an update converting tabs -> spaces.